provider/lpmessage/sender.go

```diff
@@ -39,16 +40,17 @@ type Sender struct {
 	api    SenderAPI
 	signer SignerAPI
 
-	db *harmonydb.DB
+	db     *harmonydb.DB
+	noSend bool // For early testing with only 1 lotus-provider
```

Comment:
I don't think we want a global noSend, imo it would be better to have a flag on PoSt tasks which says 'this is mock execution, don't send'